src/lakefs_spec/client_helpers.py

```diff
@@ -43,8 +62,7 @@ def commit(
 def create_branch(
     client: LakeFSClient, repository: str, name: str, source_branch: str, exist_ok: bool = True
 ) -> str:
-    """
-    Create a branch in a lakeFS repository.
+    """Create a branch in a lakeFS repository.
```

Comment:
For consistent style within this module:

```suggestion
    """Creates a branch in a lakeFS repository.
```